### PR TITLE
Added SVGBoundingBoxOptions

### PR DESCRIPTION
**Pull request by MendyBerger**
mdn/sprints/issues/3914

This is the first time I write actual content to MDN so any type of feedback (grammar, format, etc) would be greatly appreciated.

I think that we also need to add `SVGBoundingBoxOptions` to `browser-compat-data`.

### CODE REVIEW

**Comment by Ryuno-Ki** [comment]
Dear Mendy,

let me draw your attention to this line of the BETA.md:

> When you make a legitimate change to the content repo and it gets accepted, you'll also have to go in and make that change on the wiki if you want to see it live.

The alternative would be to wait until the end of the Beta phase (beginning of December), since then content should be synced (in my understanding).
In another issue, it was mentioned, that the staff team is on vacation this week, so please be a bit patient.

Thanks for your contribution!

**Comment by MendyBerger** [comment]
@Ryuno-Ki I figured since we are close enough to December that I'll make the changes here.

Thanks for letting me know that the team is on vacation, I wasn't aware of that, I guess the PR will have to wait

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Thanks @MendyBerger , this is looking pretty good. A couple of queries/nitpicks:

1. I'd like to see the default values wrapped in `<code>` elements so they show in code font.
2. I'm pretty sure that these values are boolean values, not Boolean objects, so in each case, "{{jsxref("Boolean")}}" should be changed to "boolean value" ?

**Comment by chrisdavidmills** [comment]
And @Ryuno-Ki is correct — you will also need to add this content to the wiki page version too, to stop it from being lost. The content is served from this repo as of December 14th.

**Comment by Ryuno-Ki** [comment]
>     1. I'd like to see the default values wrapped in `<code>` elements so they show in code font.

I'm sure, you meant to wrap it because it „represents a fragment of code”, not because of the look of it (e.g. the semantics are important) :-)

**Comment by chrisdavidmills** [comment]
> > ```
> > 1. I'd like to see the default values wrapped in `<code>` elements so they show in code font.
> > ```
> 
> I'm sure, you meant to wrap it because it „represents a fragment of code”, not because of the look of it (e.g. the semantics are important) :-)

Pedantry much appreciated. Yes, you are totally right ;-)

**Comment by MendyBerger** [comment]
> 1. I'd like to see the default values wrapped in `<code>` elements so they show in code font.

Updated

**Comment by MendyBerger** [comment]
> 2. I'm pretty sure that these values are boolean values, not Boolean objects, so in each case, "{{jsxref("Boolean")}}" should be changed to "boolean value" ?

I'm a bit confused, I see that some pages do use "{{jsxref("Boolean")}}" for Boolean values
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is
https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

**Comment by chrisdavidmills** [comment]
> > 1. I'm pretty sure that these values are boolean values, not Boolean objects, so in each case, "{{jsxref("Boolean")}}" should be changed to "boolean value" ?
> 
> @chrisdavidmills I'm a bit confused, I see that some pages do use "{{jsxref("Boolean")}}" for Boolean values
> https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/is
> https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener

This is possibly my fault. I used to think that the two were synonymous, but I got pulled up a while ago by a Firefox engineer about it — in most cases, you are not feeding in an actual `Boolean` object as the argument — you are feeding in a simple string value.

The spec is not actually very clear about this, for example see https://dom.spec.whatwg.org/#event-listener-capture.

But I think just saying boolean value is simpler and likely to cause less confusion.

**Comment by MendyBerger** [comment]
> And @Ryuno-Ki is correct — you will also need to add this content to the wiki page version too, to stop it from being lost. The content is served from this repo as of December 14th.

Will the changes be merged with the wiki or overridden?

**Comment by chrisdavidmills** [comment]
> > And @Ryuno-Ki is correct — you will also need to add this content to the wiki page version too, to stop it from being lost. The content is served from this repo as of December 14th.
> 
> @chrisdavidmills one more question, will the changes here be merged with the wiki or overridden?

In the beta period, the `content` repo is overwritten with the wiki content roughly about once per day. This is why we are telling people to make any changes we make here in the wiki as well just for now, so work is not lost.

After the December 14th launch date of course, this will no longer be an issue ;-)

**Comment by peterbe** [comment]
The migration has now happened. @MendyBerger is the PR ready?

**Comment by MendyBerger** [comment]
> The migration has now happened. @MendyBerger is the PR ready?

Yes, think I've addressed all concerns raised

File: files/en-us/web/api/svggraphicselement/getbbox/index.html [hunk 1]
```diff
@@ -25,6 +25,24 @@ <h2 id="Syntax">Syntax</h2>
...
 <pre class="syntaxbox notranslate">let bboxRect = <var>object</var>.getBBox();</pre>
 
+<h3 id="Return_value">Parameters</h3>
+
+<dl>
+  <dt><code><var>Options</var></code> {{experimental_inline}} {{optional_inline}}</dt>
+  <dd>An options dictionary used to control which parts of the element are included in the bounding box. The available options are:
+    <dl>
+     <dt><code>fill</code></dt>
+     <dd>A boolean value indicating that the fill should be included in the bounding box, defaults to <code>true.</code></dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/svggraphicselement/getbbox/index.html:35]:
```suggestion
     <dd>A boolean value indicating that the fill should be included in the bounding box, defaults to <code>true</code>.</dd>
```

File: files/en-us/web/api/svggraphicselement/getbbox/index.html [hunk 2]
```diff
@@ -25,6 +25,24 @@ <h2 id="Syntax">Syntax</h2>
...
+     <dt><code>stroke</code></dt>
+     <dd>A boolean value indicating that the stroke should be included in the bounding box, defaults to <code>false.</code></dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/svggraphicselement/getbbox/index.html:37]:
```suggestion
     <dd>A boolean value indicating that the stroke should be included in the bounding box, defaults to <code>false</code>.</dd>
```

File: files/en-us/web/api/svggraphicselement/getbbox/index.html [hunk 3]
```diff
@@ -25,6 +25,24 @@ <h2 id="Syntax">Syntax</h2>
...
+     <dt><code>markers</code></dt>
+     <dd>A boolean value indicating that the markers should be included in the bounding box, defaults to <code>false.</code></dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/svggraphicselement/getbbox/index.html:39]:
```suggestion
     <dd>A boolean value indicating that the markers should be included in the bounding box, defaults to <code>false</code>.</dd>
```

File: files/en-us/web/api/svggraphicselement/getbbox/index.html [hunk 4]
```diff
@@ -25,6 +25,24 @@ <h2 id="Syntax">Syntax</h2>
...
+     <dt><code>clipped</code></dt>
+     <dd>A boolean value indicating that the bounding box should be clipped, defaults to <code>false.</code></dd>
```
Comment (sideshowbarker) [inline files/en-us/web/api/svggraphicselement/getbbox/index.html:41]:
```suggestion
     <dd>A boolean value indicating that the bounding box should be clipped, defaults to <code>false</code>.</dd>
```

**Review by sideshowbarker — CHANGES_REQUESTED** [review]

**Comment by MendyBerger** [comment]
Thanks @sideshowbarker for catching it, fixed it

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/web/api/svggraphicselement/getbbox/index.html [hunk 5]
```diff
@@ -25,6 +25,24 @@ <h2 id="Syntax">Syntax</h2>
 
 <pre class="syntaxbox notranslate">let bboxRect = <var>object</var>.getBBox();</pre>
 
+<h3 id="Return_value">Parameters</h3>
```
Comment (nschonni) [inline files/en-us/web/api/svggraphicselement/getbbox/index.html:28]:
```suggestion
<h3 id="Parameters">Parameters</h3>
```

Comment (chrisdavidmills) [reply files/en-us/web/api/svggraphicselement/getbbox/index.html]: Fixed by my last update. I think we can merge this now!

**Review by chrisdavidmills — APPROVED** [review]

**Pull request merged**